server/paths/prisonerSearch.ts

```diff
@@ -1,9 +1,11 @@
 import { path } from 'static-path'
 
-const prisonerSearchPath = path('/prisoner-search/match-prisoners')
+const searchByCriteriaPath = path('/prisoner-search/match-prisoners')
+const searchByPrisonNumbersPath = path('/prisoner-search/prisoner-numbers')
 
 export default {
   prisoner: {
-    search: prisonerSearchPath,
+    searchByCriteria: searchByCriteriaPath,
+    searchByPrisonNumbers: searchByPrisonNumbersPath,
```

Comment:
This isn't used by the UI codebase, only to stub the API call - could it be misleading to have it stored here?

Reply:
I did wonder about that. I'd still consider it as part of the UI codebase; it just doesn't factor into the deployed app. But yeah, maybe best to leave it out of the `/server` code until it needs using there?